### PR TITLE
Add figure HTML to media player && padding to the bottom of the media player

### LINKED ISSUE #4647: AV: media player has no figure HTML
**Describe the bug**
The media player currently contains a figcaption but this is not contained within figure HTML. This means that items do not get announced as a group, affecting screenreader UX.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo
2. Scroll to the media player
3. Inspect
4. See figcaption is present without figure HTML:
`<figcaption class="Caption-sc-16x70so-0 gDfBZX"><span class="VisuallyHiddenText-sc-1yjwwa5-0 jFfEvX">Video caption, </span><p>This is a video block containing a video about ants</p></figcaption>`

**Expected behaviour**
Fig captions should be contained within figure HTML, as per the image with caption on the same test page (https://www.test.bbc.co.uk/news/articles/cn7k01xp8kxo)

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: RayNjeri <rachael.njeri@andela.com>

Resolves #4647 &&  #4239

**Overall change:** _Adds figure HTML to media player._

**Code changes:**

- _Adds @psammead-figure to amp and canonical media player._
- _Adds padding to the bottom of media player_(Test article for padding: **cjn0qpz3jjvo**)
- _Updated snapshots._

**Screenshots:**
Screenshot 2019-11-14 at 9 11 45 AM

Screenshot 2019-11-14 at 9 11 07 AM

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Review by rhenshaw56 — APPROVED** [review]

File: src/app/containers/MediaPlayer/index.jsx [hunk 1]
```diff
@@ -101,7 +102,7 @@ const MediaPlayerContainer = ({
   );
 
   return (
-    <>
+    <Figure>
```
Comment (sareh) [inline src/app/containers/MediaPlayer/index.jsx]: Figure shouldn't wrap the `Metadata` component - only the media player and caption.
If you can move the `<Figure>` to one row down, that'd be perfect.

Comment (RayNjeri) [reply src/app/containers/MediaPlayer/index.jsx]: Addressed and fixed.

**Review by simonsinclair — APPROVED** [review]
Nice work, Ray. UX will be very happy.

**Review by EinsteinNjoroge — APPROVED** [review]
:+1:

File: cypress/integration/pages/articles/testsForCanonicalOnly.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           .scrollIntoView();
 
         cy.get('figure')
-          .eq(1)
+          .eq(2)
```
Comment (chris-hinds) [inline cypress/integration/pages/articles/testsForCanonicalOnly.js:59]: Out of interest is this expecting to see two figure elements?

**Review by chris-hinds — APPROVED** [review]

File: cypress/integration/pages/articles/testsForCanonicalOnly.js (hunk 2 above)
Comment (RayNjeri) [reply cypress/integration/pages/articles/testsForCanonicalOnly.js:59]: Yeah, seeing we've added another figure element to wrap around the media player

**Pull request merged**